.devcontainer/Dockerfile

```diff
@@ -109,7 +113,7 @@ WORKDIR /uncrustify/uncrustify-uncrustify-0.68.1/
 RUN mkdir build
 WORKDIR /uncrustify/uncrustify-uncrustify-0.68.1/build/
 RUN cmake ..
-RUN make -sj8
+RUN MAKEFLAGS="-j $(nproc)" make -s
```

Comment:
I know, this could have been a separate PR, but YOLO

I just happened to see this was still hardcoded to the arbitrary 8 parallel jobs. It is only applicable during the building of the devcontainer, but this allows bigger github actions to actually do something significant it ever required.